website/src/blogs/install-wordpress-using-dynamic-nfs-provisioner.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Install WordPress using OpenEBS Dynamic NFS Provisioner
+author: Sai Chaithanya
+author_info: A developer who is always eager to learn, loves algorithms, maths, Kubernetes, and programming, passionate about Data Science. Enjoys playing kabaddi and traveling.
+date: 02-11-2021
+tags: OpenEBS NFS, NFS Provisioner, RWX Volume Provisioning, Dynamic NFS Provisioner
+excerpt: Guide to deploy WordPress by using OpenEBS dynamic NFS Provisioner
+not_has_feature_image: true
+---
+
+### Introduction
+  In large scale environments, storage is one of the hard things to manage, and it will be the most crucial component as it has DATA with it. OpenEBS, leading open source Cloud Native Storage, makes managing storage easier in Kubernetes environments. Mayadata, the company behind the OpenEBS project, has the vision of achieving data agility by transforming Kubernetes as a data plane. This blog is mainly for users who require ReadWriteMany(RWX) for Kubernetes applications.
+
+## OpenEBS Dynamic NFS Provisioner
+  OpenEBS Dynamic NFS Provisioner will enable you to dynamically provision ReadWriteMany(RWX/MultiNode ReaderWriter) volume on any type of Kubernetes Persistent Volumes which supports dynamic provisioning,  RWX is achieved by launching NFS Server on the Persistent Volume. In simple words, dynamic-nfs-provisioner will expose ReadWriteOnce volume as ReadWriteMany volume.
+
+**Terminology:**
+
+| Terminology          | Meaning          |
```

Comment:
Can we treat this section better by wrapping it inside code block or something. On the browser it looks congested.

Reply:
https://deploy-preview-288--openebs.netlify.app/blog/install-wordpress-using-openebs-dynamic-nfs-provisioner

Reply:
Used `<table> </table>` html table format

Reply:
Instead of using `<table> </table>` html format, can you please wrap the earlier version of the table within 3 backticks?
<img width="1440" alt="Screenshot 2021-11-10 at 11 09 09 AM" src="https://user-images.githubusercontent.com/34312966/141056581-cee37090-d7c1-4d92-a710-0470a608a818.png">

Reply:
Sure

Reply:
![image](https://user-images.githubusercontent.com/13776912/141062569-b3c45b89-9fae-4a4b-bc72-984e4df4ba4a.png)
Still I am getting something which doesn't look like table

Reply:
@mittachaitu In that case let's attach an image of the table instead of the raw table. That can be a solution.